src/core/src/core_logic/PatchInstaller.py

```diff
@@ -177,14 +179,48 @@ def install_updates(self, maintenance_window, package_manager, simulate=False):
         self.last_still_needed_packages = all_packages
         self.last_still_needed_package_versions = all_package_versions
 
+        batch_patching_stopwatch = Stopwatch(self.env_layer, self.telemetry_writer, self.composite_logger)
+        batch_patching_stopwatch.start()
+
+        installed_update_count, patch_installation_successful, maintenance_window_batch_cutoff_reached = self.install_patches_in_batches(all_packages, packages, package_versions, maintenance_window, package_manager)
+
+        batch_patching_stopwatch.stop()
+
+        # Check for updates available to install. After installing patches in batches, there might be still some packages available for installation. These packages should be:
+        # (a) Due to not enough remaining time in maintenance window to install the updates in batch, the batch patching is stopped and hence some updates are not installed in batches
+        # (b) Those which were failed while installing in batches
+        # (c) Some new packages available in this brief time when batch patching was being done
+        packages, package_versions = self.get_remaining_packages_to_install(package_manager)
+
+        batch_processing_perf_log  = {"Installed patches count in batch processing": str(installed_update_count), "number of remaining packages to install": str(len(packages)), 
```

Comment:
Similar comment to the one above, log statement uses patches and packages to indicate the same thing.

Reply:
Done. Using packages instead of batches.